src/main/common/filter.c

```diff
@@ -65,6 +65,64 @@ FAST_CODE float pt1FilterApply(pt1Filter_t *filter, float input)
     return filter->state;
 }
 
+// PT2 Low Pass filter
+
+float pt2FilterGain(float f_cut, float dT)
+{
+    float RC = 1 / ( 3.10755f * M_PIf * f_cut); // cutoff 2 * 1.553774 compared to PT1
+    // maths is 2 / sqrt( (2^(1 / n) - 1) ) where n is order
```

Comment:
You can probably even put this into a define and then evaluate for the order of filter - the compiler will precalculate all constant expressions and use the calculated values.

Reply:
Will do. I just used the constant because I wasn't sure how to write the code and then I'd have to test it :-)

Reply:
Maths is now fully implemented, thanks @mikeller for your help with that.